File: Q11/main.c
#include<stdio.h>
int main(){
    int row,col;
    scanf("%d%d",&row,&col);

    int arr[row][col];

    for(int i=0;i<row;i++){
        for(int j=0;j<col;j++){
            scanf("%d",&arr[i][j]);
        }
    }
    for(int i=0;i<row;i++){
        int min = arr[i][0];
        for(int j=0;j<col;j++){
            if(arr[i][j]<min){
                min = arr[i][j];
            }
        }
        printf("%d\n",min);
    }
    return 0;
}

File: Q11/main.py
row = int(input())
col = int(input())

arr = []

for i in range(row):
    row_data = list(map(int, input().split()))
    arr.append(row_data)

for i in range(row):
    min_value = arr[i][0]

    for j in range(col):
        if arr[i][j] < min_value:
            min_value = arr[i][j]

    print(min_value, end=" ")